### ISSUE TITLE
curl crashes kerberos missing (Missing framework: Kerberos)

### ISSUE THREAD

**Issue by Tanja-4732** [darlinghq/darling#1431]
## TL;DR

`curl` always crashes, Kerberos seems to be missing

## Details

**Framework**
We're missing Kerberos

**Test Application**
I tried to run `curl http://example.org` as shown in the documentation,  
but `curl` alone also crashes the exact same way.

**Steps To Reproduce**
1. Build & install Darling
2. `darling shell zsh`
3. `curl` (optionally with arguments)
4. Kaboom :crying_cat_face: 

**System Information**
I'm running 6.1.51-1-MANJARO x86_64.

| Software | Version |
| --- | --- |
| Darling | aa9eddbc0
`curl`|The bundled one
`zsh`|The bundled one
Kerberos| None :crying_cat_face: 

**Error message**

```zsh
darling on  master via △ v3.27.4 took 6s 
❯ darling shell zsh
@tanja-pc darling % curl
dyld: Library not loaded: /System/Library/Frameworks/GSS.framework/Versions/A/GSS
  Referenced from: /System/Library/Frameworks/Kerberos.framework/Versions/A/Kerberos
  Reason: image not found
abort_with_payload: reason: dyld: No shared cache present
Library not loaded: /System/Library/Frameworks/GSS.framework/Versions/A/GSS
  Referenced from: /System/Library/Frameworks/Kerberos.framework/Versions/A/Kerberos
  Reason: image not found; code: 1
zsh: abort (core dumped)  curl
@tanja-pc darling % 
```

**Comment by facekapow** [comment]
That's strange, Kerberos should've been installed alongside `curl`. Did you use a `COMPONENTS` option when configuring the build or is this a normal full build of Darling?

Try reinstalling and see if that fixes it.
